source/Classes/1.DscResources/07.nxService.ps1

```diff
@@ -35,18 +35,25 @@ class nxService
     {
         $nxService = Get-nxService -Name $this.Name
         $currentState = [nxService]::new()
+        if ([string]::IsNullOrEmpty($nxService.Name))
+        {
+            if ($null -ne $this.Enabled -and -not $this.Enabled -and $null -ne $this.State -and $this.State -eq [nxServiceState]::Stopped)
```

Comment:
$this.State is a string while we are comparing it to an nxServiceState. Is PowerShell ok with that? Just asking; not sure

Reply:
I couldn't find a lot of information online. I found [one stackoverflow thread](https://stackoverflow.com/a/57834292) that says this is how you compare strings with enums. I went back and manually tested again and it's working.